### PR TITLE
feat: mantra integration

### PR DESCRIPTION
**Pull request by Canestin**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - add mantra currency

### ❓ Context

- **JIRA link**: https://ledgerhq.atlassian.net/browse/LIVE-14611

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by mcayuelas-ledger — APPROVED** [review]
LGTM for @LedgerHQ/live-hub

**Review by bharamboure-ledger — APPROVED** [review]

**Comment by mcayuelas-ledger** [comment]
Can you rebase your PR plz ? :)

**Review by bharamboure-ledger — DISMISSED** [review]

File: libs/coin-modules/coin-cosmos/src/chain/Mantra.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  stakingDocUrl: string;
+  unbondingPeriod: number;
+  prefix: string;
+  validatorPrefix: string;
+  // Provided by coin config
+  ledgerValidator!: string;
+  lcd!: string;
+  constructor() {
+    super();
+    this.stakingDocUrl = "";
```
Comment (hedi-edelbloute) [inline libs/coin-modules/coin-cosmos/src/chain/Mantra.ts:13]: We need to add the doc url here

**Review by hedi-edelbloute — DISMISSED** [review]

File: libs/coin-modules/coin-cosmos/src/chain/Mantra.ts (hunk 1 above)
Comment (Canestin) [reply libs/coin-modules/coin-cosmos/src/chain/Mantra.ts]: https://github.com/LedgerHQ/ledger-live/pull/8205/files#diff-ac58aed97bc6283523a1a1299b2f73d88100f93300cb1027cbc61709934f01c6R13

**Review by bharamboure-ledger — APPROVED** [review]

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**